### DISCUSSION TITLE
Is this the JDBC driver that is used by default for Glue ETL MySQL connections?

### DISCUSSION THREAD

**Discussion by GregCKrause** [awslabs/aws-mysql-jdbc#642] [Q&A]
Or does Glue use a different JDBC driver by default for MySQL connections?

**Comment by davecramer** [comment]
I have no idea what Glue is using, however this driver has been deprecated and we will no longer be providing support for it.
Please use https://github.com/aws/aws-advanced-jdbc-wrapper moving forward
